### DISCUSSION TITLE
modules error

### DISCUSSION THREAD

**Discussion by JtheLyn** [TheLastBen/fast-stable-diffusion#1368] [Q&A]
```
Traceback (most recent call last):
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/webui.py", line 13, in <module>
    from modules.call_queue import wrap_queued_call, queue_lock, wrap_gradio_gpu_call
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/call_queue.py", line 7, in <module>
    from modules import shared
  File "/content/gdrive/MyDrive/sd/stable-diffusion-webui/modules/shared.py", line 390, in <module>
    "img2img_background_color": OptionInfo("#ffffff", "With img2img, fill image's transparent parts with this color.", ui_components.FormColorPicker, {}),
AttributeError: module 'modules.ui_components' has no attribute 'FormColorPicker'
```
![img-2023-01-13-01-20-32](https://user-images.githubusercontent.com/53907742/212135718-738b36b3-e12f-40fe-a12a-a41178fa08aa.png)

I keep getting this error in the last cell. What does it mean and how can I fix it?

**Comment by TheLastBen** [comment]
Remove/rename the SD folder and use the latest colab

**Reply by JtheLyn** [reply]
A fresh install works, but the same error pops up every now and then as well as some other errors. Thankfully my drive has unlimited space, but it does get troublesome. I keep getting this error even after multiple fresh installations.
![image](https://user-images.githubusercontent.com/53907742/212548516-29a1879d-9c6b-457d-9e61-507eacbe98a5.png)

**Comment by Miquel-code** [comment]
Hello **JtheLyn**, just like you I've been having the same error by the last two weeks. I recommend you delete the **_SD_** folder and redownload it again.

Don't forget to save your '_Models_', '_ckpt_' files and outputs before. 🛠

**Comment by TheLastBen** [comment]
It's not your sd folder, it's the colab you're using, it's outdated, use the links from the main page of this repo
